### PR TITLE
Send AccountingPointcharacteristics_MarketDocument actor message on connect

### PR DESCRIPTION
**Pull request by cmgrd**
Send RSM-022 actor message on connect.

Still missing:
- XML/query model/query is not complete. This needs to be revisited later when we know the final XML format and when we're certain on the read/query approach.

### CODE REVIEW

**Review by thbaan — DISMISSED** [review]

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/DataAccess/IDbConnectionFactory.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+namespace Energinet.DataHub.MeteringPoints.Infrastructure.DataAccess
+{
+    /// <summary>
+    /// Factory for creating database connections
+    /// </summary>
+    public interface IDbConnectionFactory
+    {
+        /// <summary>
+        /// Returns and existing open connection if any, or creates a new open connection
```
Comment (Korgath) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/DataAccess/IDbConnectionFactory.cs:25]: Nit: Typo

File: source/Energinet.DataHub.MeteringPoints.Application/Queries/MeteringPointByGsrnQuery.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using MediatR;
+
+namespace Energinet.DataHub.MeteringPoints.Application.Queries
+{
+    public class MeteringPointByGsrnQuery : IRequest<ConsumptionMeteringPoint>
```
Comment (kft) [inline source/Energinet.DataHub.MeteringPoints.Application/Queries/MeteringPointByGsrnQuery.cs:20]: Would it be more intention revealing if we named it `GetMeteringPointByGsrn` or `RetrieveMeteringPointByGsrn` ?

**Review by kft — APPROVED** [review]
LGTM! A single nit :-)

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/DataAccess/IDbConnectionFactory.cs (hunk 1 above)
Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/DataAccess/IDbConnectionFactory.cs]: Fixed.

File: source/Energinet.DataHub.MeteringPoints.Application/Queries/MeteringPointByGsrnQuery.cs (hunk 2 above)
Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.Application/Queries/MeteringPointByGsrnQuery.cs:20]: Perhaps. But isn't that implicit from the Query suffix?

**Review by kft — APPROVED** [review]

**Review by thbaan — DISMISSED** [review]

**Review by thbaan — APPROVED** [review]

**Review by kft — APPROVED** [review]

**Pull request merged**